ls1intum/Artemis
# `Programming exercises`: Improve layout of taskbar

### Checklist
#### General


#### Client

### Motivation and Context


The task bar uses two table cells. One for "Task"/"Teilaufgaben" and the other for all the task icons. We want to support up to 15 task icons, so they should get the majority of space compared to the one word at the start of the bar.

### Description

This PR shrinks the cell of the one word to its minimum size to give the task icons more of the available space. This especially reduces the unused space between the one word and the first task icon.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise

1. Log in to Artemis
2. Create the exercise/Find one with some tasks in the problem statement
3. Look at the task bar to see the icons that are more evenly spaced out

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots


Before:
<img width="1727" alt="Bildschirmfoto 2023-06-18 um 16 09 51" src="https://github.com/ls1intum/Artemis/assets/38322605/5a96d2d8-d3fc-43b0-82e8-32ef70ab4f66">
<img width="564" alt="Bildschirmfoto 2023-06-18 um 16 10 00" src="https://github.com/ls1intum/Artemis/assets/38322605/fa21faec-d523-4834-b048-0731af1b8809">

With this PR:
<img width="1727" alt="Bildschirmfoto 2023-06-18 um 16 09 38" src="https://github.com/ls1intum/Artemis/assets/38322605/14f694c6-c889-4d46-9b35-4870e6e7e9d1">
<img width="564" alt="Bildschirmfoto 2023-06-18 um 16 10 13" src="https://github.com/ls1intum/Artemis/assets/38322605/e5345a4a-589f-4dc4-92fe-33dc02f53209">
